packages/common/base/src/blockchain/utils/isBlockchainIncludingTestnets.ts

```diff
@@ -0,0 +1,10 @@
+import { BLOCKCHAINS_INCLUDING_TESTNETS, Blockchain, BlockchainIncludingTestnet } from "../types";
+
+export function isBlockchainIncludingTestnets<T extends BlockchainIncludingTestnet = BlockchainIncludingTestnet>(
+    value: unknown,
+    expectedBlockchain?: T
+): value is T {
```

Comment:
are generics really necessary here? feels like this is the same but simpler
```suggestion
export function isBlockchainIncludingTestnets(value: unknown, expectedBlockchain?: BlockchainIncludingTestnet): value is BlockchainIncludingTestnet {
```

or, is this used anywhere in our SDKs? Might just not want to include this

Reply:
see below, if we keep it as the building block for all the `isSOMECHAIN` checks, then it should keep the generic